ffi/threshold.h

```diff
@@ -41,22 +41,6 @@ typedef struct {
   int len;
 } Buffer;
 
-/**
```

Comment:
Since this is removed, the `BlindedMessage` struct in Rust should also be removed